### ISSUE TITLE
node/perf_hooks_test.ts fail

### ISSUE THREAD

**Issue by Sorikairox** [denoland/deno_std#2240]
**Describe the bug**
When running `deno test` and `deno test --unstable` in the repository, I get the following errors : 

```shell
error: TS2339 [ERROR]: Property 'toJSON' does not exist on type 'Omit<Performance, "clearMeasures" | "getEntries" | "getEntriesByName" | "getEntriesByType"> & { eventLoopUtilization: any; nodeTiming: Record<string, string>; timerify: any; timeOrigin: any; }'.
    assertEquals(perfHooks.performance.toJSON, performance.toJSON);
                                       ~~~~~~
    at file /deno_std/node/perf_hooks_test.ts:13:40

TS2339 [ERROR]: Property 'toJSON' does not exist on type 'Omit<Performance, "clearMeasures" | "getEntries" | "getEntriesByName" | "getEntriesByType"> & { eventLoopUtilization: any; nodeTiming: Record<string, string>; timerify: any; timeOrigin: any; }'.
    assertEquals(perfHooks.performance.toJSON, performance.toJSON);
                                                           ~~~~~~
    at file /deno_std/node/perf_hooks_test.ts:13:60

TS2339 [ERROR]: Property 'toJSON' does not exist on type 'Omit<Performance, "clearMeasures" | "getEntries" | "getEntriesByName" | "getEntriesByType"> & { eventLoopUtilization: any; nodeTiming: Record<string, string>; timerify: any; timeOrigin: any; }'.
    perfHooks.performance.toJSON();
                          ~~~~~~
    at file /deno_std/node/perf_hooks_test.ts:18:27

TS2339 [ERROR]: Property 'toJSON' does not exist on type 'Omit<Performance, "clearMeasures" | "getEntries" | "getEntriesByName" | "getEntriesByType"> & { eventLoopUtilization: any; nodeTiming: Record<string, string>; timerify: any; timeOrigin: any; }'.
    performance.toJSON();
                ~~~~~~
    at file /deno_std/node/perf_hooks_test.ts:29:17

```

**Steps to Reproduce**

Run `deno test` or `deno test --unstable`

**Expected behavior**

Not throw any error.

**Environment**

MacOS Catalina 10.15.7
deno 1.21.3 (release, x86_64-apple-darwin)
v8 10.0.139.17
typescript 4.6.2

- std version: up to date with main branch

**Comment by bartlomieju** [comment]
This is expected - tests on CI run on latest `canary` version. The error you hit is coming from an API that is available in the latest `canary`, but wasn't available in v1.21.3 (but will be available in v1.22 on Wednesday). There's not much that can be done about it, besides adding `@ts-ignore` directives locally above failing lines.

**Comment by Sorikairox** [comment]
Thanks for the explanation!
